src/components/Events.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
I'm a bit confused about what "Events" is vs EventList, and EventListItem, and EventMap. Is this an individual event page?

Reply:
ah, it's the container for an event. does the endpoint events need to exist without passing in an event ID?

Reply:
👍 this is basically a container to hold either the map or the list, along with other components that show in both views (e.g. the toggle buttons). I could probably pick a better name.